examples/seq2seq/README.md

```diff
@@ -188,22 +188,23 @@ Note, that depending on the used model additional language-specific command-line
 * T5 requires:
 
    ```
+       --source_lang en \
+       --target_lang ro \
```

Comment:
T5 does not requires that, technically. It's required for the data preprocessing mostly.

Reply:
yup of course, but my current tradeoff (simplicity vs redundancy) makes them required (I raise a ValueError if these parameters aren't set), we can discuss the tradeoff but the doc is currently correct in this state.

Reply:
Yes, and I like that they are needed arguments. But with this change they are not "additional language-specific command-line arguments are sometimes required" as said above. I would remove the paragraph about mBART before, or say it requires custom source/target language codes like `en_XX` instead of `en` and just mention the `source_prefix` for T5 here.